.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,50 @@
+# Notes:
+# /__w/openssl is the path that github bind-mounts into the container so the ci
+# filesystem for this job can be reached.  Please note that any changes made to
+# this job involving file system paths should be made prefixed with, or relative
+# to that directory
+name: Interoperability tests with GnuTLS and NSS
+on:
+  schedule:
+    - cron: '0 6 * * *'
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: docker.io/fedora:39
+      options: --sysctl net.ipv6.conf.lo.disable_ipv6=0
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [gnutls, nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    steps:
+      - uses: actions/checkout@v4
+      - name : Install needed tools
+        run: |
+          dnf -y install perl gcc rpmdevtools dnf-utils make tmt-all beakerlib \
+                 fips-mode-setup crypto-policies-scripts
+      - name: install interop tests
+        run: |
+          cd /__w/openssl/openssl
```

Comment:
Is this prefix not available as an environment variable? I assume GitHub reserves the right to change this.

Reply:
Unfortunately no, and the docs don't mention anything about a workdir env variable.  You can see here:
https://github.com/openssl/openssl/actions/runs/7151995569/job/19476853336
That in the Initialize container step, github starts the container with the --workdir setting to set that path, but never sets a corresponding environment variable for us to fetch that value from.

We could manually set it and reference it, but that doesn't help if github changes that path.  I'm guessing they just haven't thought about the implications of doing so.

Only way I could see around this would be to explicitly add a bind mount in the volumes section, but that comes with its own issues, notably if github changes the runners physical path, it still breaks, and the fact that we would have two bind mounts to the physical path in the container, which, while allowed, would seem odd and prone to odd behaviors I think

Reply:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

What about `$GITHUB_WORKSPACE`?

Adding `export` to a workflow might be a useful aid to get a sense of what variables are available.

Reply:
oh, yes, that does map properly, I'll use that and add an export step to dump the environment

Reply:
Updated to use GITHUB_WORKSPACE